packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -549,6 +556,20 @@ export function signInActor({ services }: SignInMachineOptions) {
         shouldSetupTOTP: (_, event): boolean => {
           return isExpectedChallengeName(getChallengeName(event), 'MFA_SETUP');
         },
+        /* START ISSUE 3767 - Temporary fix. Will change to handle 'SELECT_MFA_TYPE' */
+        isSelectMFATypeChallenge: (_, event): boolean => {
+          const isSelectMfaTypeChallenge = isExpectedChallengeName(
+            getChallengeName(event),
+            'SELECT_MFA_TYPE'
+          );
+          if (isSelectMfaTypeChallenge) {
+            console.error(
+              "ERROR: 'SELECT_MFA_TYPE' is not supported by Amplify UI yet. \n Please use only one MFA type or set up the MFA selector using this documen: https://docs.amplify.aws/lib/auth/mfa/q/platform/js/#advanced-use-cases/"
```

Comment:
In the error message, I'm trying to communicate
1) 'SELECT_MFA_TYPE' is not supported by Amplify UI, but users can implement it on their own using the API by following the link
2) 'SELECT_MFA_TYPE' will be supported by Amplify UI sometime in the future because we've got the feature request issue

Any thought?